phpdocker/php-fpm/entrypoint.sh

```diff
@@ -11,4 +11,7 @@ npm install
 echo -e "\nGenerate assets (JS/CSS)"
 npm run-script build
 
+echo -e "\nApply DB migrations if needed"
+bin/console doctrine:migrations:migrate --em=default
```

Comment:
Should be add --no-interactive to make sure it's executed without asking the user for confirmation? If not we can see this text:

```
php-fpm_1        |  WARNING! You are about to execute a migration in database "goteo_oauth" that could result in schema changes and data loss. Are you sure you wish to continue? (yes/no) [yes]:
```

Reply:
I've applied what you suggested just in case. You might be right and it wasn't being executed, but haven't checked it. Thanks!

Reply:
Well, the message was shown when executing ```docker-compose up``` but i checked that the tables were being created still, it was just to make sure that it was being executed still